src/locales/hi/common.json

```diff
@@ -0,0 +1,9 @@
+{
+    "header":{
+        "category":{
+            "men":"पुरुष",
```

Comment:
Nice work! This should also contain words for login / logout etc.

Reply:
yes, As progress, will be changed here.